uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/APIWrapper/DefaultLimitRemoverTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use Civi\Test\HeadlessInterface;
+use Civi\Test\TransactionalInterface;
+
+/**
+ * Class CRM_HRCore_APIWrapper_DefaultLimitRemoverTest
+ *
+ * @group headless
+ */
+class CRM_HRCore_APIWrapper_DefaultLimitRemoverTest extends \PHPUnit_Framework_TestCase implements HeadlessInterface, TransactionalInterface {
+
+  private $defaultLimitRemoverObj;
+
+  private $testParameters = [];
+
+  public function setUpHeadless() {
+    return \Civi\Test::headless()
+      ->installMe(__DIR__)
+      ->apply();
+  }
+
+  public function setUp() {
+    $this->defaultLimitRemoverObj = new CRM_HRCore_APIWrapper_DefaultLimitRemover();
+
+    $this->testParameters['params'] = [
+      'action' => 'get',
+      'entity' => 'contact',
+      'id' => 1,
+    ];
+  }
+
+  public function testFromApiInputWithLimitSetInParameters() {
```

Comment:
the problem here is not the size of the name, but the syntax we're using. I personally prefer snake case for test names (`public function default_limit_should_not_be_applied_if_a_specific_limit_has_been_set`), but unfortunately, we're following civi's convention here. Maybe we can change this in the future